R/pkg/R/functions.R

```diff
@@ -3120,6 +3147,24 @@ setMethod("arrays_overlap",
             column(jc)
           })
 
+#' @details
+#' \code{arrays_zip}: Returns a merged array of structs in which the N-th struct contains all N-th
+#' values of input arrays.
+#'
+#' @rdname column_collection_functions
+#' @aliases arrays_zip arrays_zip,Column-method
+#' @note arrays_zip since 2.4.0
+setMethod("arrays_zip",
+          signature(x = "Column"),
+          function(x, ...) {
```

Comment:
should add in doc that `...` are Column for this function
https://github.com/huaxingao/spark/blob/87100ec766acf647e91ab14450f1c33c0f929d52/R/pkg/R/functions.R#L198